motoon-eg/healthy
# feat: [HLTH-01] Add splash screen 🎯

### Description

### Screenshot
| | |
| -- | -- |
| | |

### How to test
1. Open the app
2. Check the updated splash screen
